Procfile

```diff
@@ -3,7 +3,7 @@
 ###############################
 
 # Procfile for development using the new threaded worker (scheduler, twitter stream and delayed job)
-web: bundle exec rails server -p ${PORT-3000} -b ${IP-0.0.0.0}
+web: bundle exec rails server -p ${PORT} -b ${IP-0.0.0.0}
```

Comment:
It think it would not cause any harm to keep the fallback for exotic 12factor aware setups we are not ware of. It should not be an issue for anyone using foreman, docker, or any deployment method I am aware of. However one could probably have a setup which uses a `Procfile` an not have updated the `.env` file.